testing/surf/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=surf
+pkgver=2.0
+pkgrel=0
+pkgdesc="surf is a simple web browser based on WebKit2/GTK+."
+url="https://surf.suckless.org/"
+arch="all"
+install=""
+license="MIT"
+
+depends="xprop"
+makedepends="webkit2gtk-dev"
+subpackages="$pkgname-doc"
+options="!check"
+
+source="$pkgname-$pkgver.tar.bz2::https://git.suckless.org/$pkgname/snapshot/$pkgname-$pkgver.tar.bz2"
+
+builddir="$srcdir/"$pkgname-$pkgver
+
+unpack() {
+        default_unpack
+}
+
+build() {
+        cd "$builddir"
+	make
+}
+
+package() {
+        cd "$builddir"
+
+	mkdir -p "$pkgdir"/usr/bin
+	mv $pkgname "$pkgdir"/usr/bin
+}
+
+doc() {
```

Comment:
You don't need a custom doc subpackage function. The default doc subpackage function will (among other things) move all files from `/usr/share/doc` and `/usr/share/man` to its `$subpkgdir`. Besides it will take care of compressing man pages.

Reply:
That is true that it only processes directories, but I read the source code and it doesn't handle GitHub readmes and other files containing useful information.

Reply:
Sure, that is in fact correct but you can still install these other files from the `package()` function and they will be moved to the doc subpackage by the default doc subpackage function.

Many aports do that e.g. [cryptsetup](https://github.com/alpinelinux/aports/blob/master/main/cryptsetup/APKBUILD#L47-L49).